includes/spam.php

```diff
@@ -0,0 +1,139 @@
+<?php
+/**
+ * Code related to spam detection and prevention.
+ */
+// Constants. Define these in wp-config.php to override.

```

Comment:
Do we for sure want to introduce new constants for this or create a function with the value filtered so we can offer snippets?

Asking since snippets seem better organized for customizations and constants end up piling up together in the same spot of wp-config.php

I'm still good with the constant usage here, just asking the Q in case it makes sense one way or the other.

Reply:
Earlier those constants were used in both the get and update functions, so I needed them defined outside of the functions. But in the current implementation, they are only used in one function each. So making them params could work. I like it.

Then you could unhook our functions and add your own call backs that called the same functions, but with the parameters you wanted.

I still have a feeling that these 2 numbers might be used in different contexts as we build on this. But we can code this how it's best for now and cross that bridge when we get there. Constants are difficult to set up and manage and should be avoided if possible.

Reply:
The constants are used across different functions, so leaving as constants for now. May refactor later.